services/explorer-ui/src/api/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from "./contract";
+export * from "./block";
+export * from "./tx-effects";
```

Comment:
This should be singular like the other files. (Or all of them should be plural)

`tx-effect`